### ISSUE TITLE
Clarify tooltip of Back and Undo buttons

### ISSUE THREAD

**Issue by laurent22** [laurent22/joplin#12124]
- Back: Rename to "Go back one page" (or something similar - maybe check how it's named in other apps)

- Undo: Not sure how it should be named? Maybe "Undo last change"?
